adapters/polychain-monsters/index.js

```diff
@@ -0,0 +1,115 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const BigNumber = require("bignumber.js");
+const Ethereum = require("../../sdk/EVMC");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const { AVAILABLE_PROTOCOLS } = require("../../sdk/util");
+
+class PolychainMonsters {
+    constructor() {
+        this.name = "polychain-monsters";
+        this.token = "0x1796ae0b0fa4862485106a0de9b654efe301d0b2";
+        this.protocol = AVAILABLE_PROTOCOLS.BSC;
+        this.block = 8482954;
+        this.contract = "0x85F0e02cb992aa1F9F47112F815F519EF1A59E2D";
+        this.events = ["Transfer"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+        this.sdk = new Ethereum(this);
+    }
+
+    run = async () => {
+        const s = await this.getSymbol();
+        this.sdk = this.loadSdk();
+        this.symbol = s;
+        await this.sdk.run();
+    };
+
+    loadSdk = () => {
+        return new Ethereum(this);
+    };
+    getSymbol = async () => {
+        const resp = await axios.get(
+            `${URL}/token-metadata?key=${KEY}&token_address=${this.token}&protocol=${this.protocol}`,
+            {
+                headers: {
+                    "User-Agent":
+                        "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4619.141 Safari/537.36",
+                },
+            },
+        );
+        const symbol = resp.data;
+        return symbol;
+    };
+    getPrice = async timestamp => {
+        const resp = await axios.get(
+            `${URL}/token-price?key=${KEY}&token_address=${this.token}&protocol=${this.protocol}&timestamp=${timestamp}`,
+            {
+                headers: {
+                    "User-Agent":
+                        "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4619.141 Safari/537.36",
+                },
+            },
+        );
+        return resp.data;
+    };
+    stop = async () => {
+        this.sdk.stop();
+    };
+
+    getBuyer = async event => {
+        const buyer = event.returnValues.owner;
+        if (event.event === "NameRenewed") {
```

Comment:
You are parsing `Transfer` events only, so event called `NameRenewed` will never be found.

Reply:
Good catch, thanks!